addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -763,20 +762,15 @@
 <template id='lesson_card' name="Lesson Card">
     <div class="card w-100 o_wslides_lesson_card mb-4">
             <t t-if="slide.is_new_slide and not channel_progress[slide.id].get('completed')" t-call="website_slides.course_card_information"/>
+        <t t-set="default_slide_background" t-value="website.image_url(channel, 'image_1024') if channel.image_1024 else '/website_slides/static/src/img/channel-%s-default.jpg' % ('training' if channel.channel_type == 'training' else 'documentation')"/>
```

Comment:
I'm wondering if we should not in the future replace the image field of the slide by the website cover mixin. What do you think ?

Reply:
Ok we can discuss this with Luc tomorrow. There are pros and cons but either way I think that it could be cool to add at least a way to edit image of the slides on frontend